web/src/components/AssertionCheckRow/AssertionCheckRow.tsx

```diff
@@ -1,22 +1,24 @@
-import {useMemo} from 'react';
 import {difference} from 'lodash';
+import {useMemo} from 'react';
 import OperatorService from '../../services/Operator.service';
 import {ISpanAssertionResult} from '../../types/Assertion.types';
 import {ISpan} from '../../types/Span.types';
-import * as S from './AssertionCheckRow.styled';
 import AttributeValue from '../AttributeValue';
+import * as S from './AssertionCheckRow.styled';
 
 interface IAssertionCheckRowProps {
   result: ISpanAssertionResult;
   span: ISpan;
   assertionSelectorList: string[];
+
   getIsSelectedSpan(spanId: string): boolean;
+
```

Comment:
Curious, why the empty line?

Reply:
prettier add those spaces to methods inside interfaces